server/bin/pbench-backup-tarballs.py

```diff
@@ -251,12 +250,10 @@ def backup_to_local(
                 )
 
                 # remove the copied md5 file from backup
-                bmd5_file = os.path.join(
-                    backup_controller_path, "{}.md5".format(resultname)
-                )
-                if os.path.exists(bmd5_file):
+                bmd5_file = backup_controller_path / "{}.md5".format(resultname)
```

Comment:
Can you also change this to use: `f"{resultname}.md5"`?

Reply:
ditto